docs/includes/usage-examples/DeleteOneTest.php

```diff
@@ -25,16 +26,36 @@ public function testDeleteOne(): void
             ],
         ]);
 
-        // begin-delete-one
+        // begin-eloquent-delete-one
         $deleted = Movie::where('title', 'Quiz Show')
             ->orderBy('_id')
             ->limit(1)
             ->delete();
 
         echo 'Deleted documents: ' . $deleted;
-        // end-delete-one
+        // end-eloquent-delete-one
 
         $this->assertEquals(1, $deleted);
-        $this->expectOutputString('Deleted documents: 1');
+
+        Movie::insert([
+            [
+                'title' => 'Quiz Show',
+                'runtime' => 133,
+            ],
+        ]);
+
+        // begin-qb-delete-one
+        $deleted = DB::table('movies')
+            ->where('title', 'Quiz Show')
+            ->orderBy('_id')
```

Comment:
Currently, the `sort` option is ignored. This is not supported by the server [PHPORM-169](https://jira.mongodb.org/browse/PHPORM-169)

Reply:
done!